Apromore-Custom-Plugins/User-Admin-Portal-Plugin/src/main/java/org/apromore/plugin/portal/useradmin/UserAdminController.java

```diff
@@ -777,6 +777,7 @@ private void setSelectedUsers(Set<User> users) {
         }
         updateAssignedRoleModel(users);
         updateAssignedGroupModel(users);
+        isUserDetailDirty = false; // extra defensive, ensure dirty is not set by field's setValue
```

Comment:
Would it make sense to remove the duplicate `isUserDetailDirty = false;` on line 730?  It just seems odd to set it twice in the same method.  The convention seems to be to set the dirty flags at the end of methods that mutate or update the current user or group.